index.html

```diff
@@ -73,7 +284,64 @@ <h5>LET'S CONNECT</h5>
             </li>
           </ul>
         </div>
+        <button class="see-link">
+          <a href="#">Get my resume</a>
+        </button>
       </div>
     </main>
+    <aside>
+      <div class="lang-header">
+        <div>
+          <h5>Languages</h5>
+        </div>
+        <div>
+          <img src="./img/Union.svg" alt="down arrow" />
+        </div>
+      </div>
+      <ul class="language-container">
+        <li class="lang">
+          <div>
+            <img src="./img/lang1.svg" alt="javascript" />
+          </div>
+          <div>
+            <h6 class="lang-text">JavaScript</h6>
+          </div>
+        </li>
+        <li class="lang">
+          <div>
+            <img src="./img/lang2.svg" alt="html" />
+          </div>
+          <div>
+            <h6 class="lang-text">HTML</h6>
+          </div>
```

Comment:
- [ ] Nice work you did with this section of the tech stack listing 👏 . However, kindly check my comparison of the figma design and your current design on the need to adjust the fonts to be consistent and have a better user experience view by sticking to the figma font sizes 👍 

## Your view :
![image](https://user-images.githubusercontent.com/50990925/221326173-b8d91a81-b22e-4481-88fd-dc0ad934c6ec.png)
 
 ## Figma:
 
![image](https://user-images.githubusercontent.com/50990925/221326203-9b8eb5ff-d6c7-47bb-a09a-754c4f197c92.png)